src/data/WorkSchedData.js

```diff
@@ -0,0 +1,169 @@
+import React, { Component } from 'react';
+import { Table, Container, Row, Col, ListGroup, ListGroupItem } from 'reactstrap';
+
+const timeLine = () => {
+    return (
+            <div>
+                <ListGroup>
```

Comment:
Please improve this ListGroup

Reply:
Okay I changed it to a class and removed the extra ListGroupItems.